### DISCUSSION TITLE
I18n: Multiple domains with same default locale not allowed

### DISCUSSION THREAD

**Discussion by SalmenBejaoui** [vercel/next.js#65714] [Help]
### Summary

I need to configure multiple domains (local, stage and production) with the same default locale but seems not allowed in next.config.js. Am I missing something? Thank you

<img width="1140" alt="image" src="https://github.com/vercel/next.js/assets/30017004/ffc6e965-e8a1-4a27-abe5-4e13e7342a75">

### Additional information

```js
Node.js v18.19.1

Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.5.0: Wed May  1 20:12:58 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T6000
Binaries:
  Node: 18.19.1
  npm: 10.2.4
  Yarn: 1.22.19
  pnpm: N/A
Relevant Packages:
  next: 14.1.0
  eslint-config-next: 14.1.0
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.3.3
Next.js Config:
  output: standalone
```

**Comment by SalmenBejaoui** [comment]
I discovered that it's an expected behavior https://github.com/vercel/next.js/pull/22337: I managed this directly in the CICD pipeline modify the domains.

**Reply by rishadAlamy** [reply]
how ?
